samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -189,13 +190,20 @@ const StorybookDecorator = (story, config) => {
 
 	return (
 		<Agate
+			key={skinKnobs.skin}
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
 			locale={locale}
 			{...skinKnobs}
 			skinVariants={boolean('night mode', Config) && 'night'}
-			accent={color('accent', (!newSkin && accentFromURL ? accentFromURL : defaultColors[currentSkin].accent), Config.groupId)}
-			highlight={color('highlight', (!newSkin && highlightFromURL ? highlightFromURL : defaultColors[currentSkin].highlight), Config.groupId)}
+			accent={useSkinDefaultStyles && !allSkins ?
+				defaultColors[skinKnobs.skin].accent :
+				color('accent', (!newSkin && accentFromURL ? accentFromURL : defaultColors[currentSkin].accent), Config.groupId)
```

Comment:
Why didn't we use `skinKnobs.skin` in the first place instead of `currentSkin`?

Reply:
`currentSkin` is always `Gallium` now.